### PR TITLE
Feature add fallback values node names

### PR DESCRIPTION
**Pull request by Matwog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
+
 [![CircleCI](https://circleci.com/gh/signavio/BPMN-Sol/tree/master.svg?style=svg&circle-token=ef21a374de1b68ab12004774d76abcb3512868f3)](https://circleci.com/gh/signavio/BPMN-Sol/tree/master)
 
 `BPMN-Sol` is a compiler to convert BPMN 2.0 xml to solidity smart contract.
 
+## Changes in the new version
+
+Previously, with the version `v0.0.3` of the `bpmn-sol` it was not possible to compile a business process model with multiple XOR gateways and multiple script tasks. When tried to compile such a diagram, the user would get a compilation error.
+
+Now with the new version `v1.0` the functionality to compile a business process model with multiple script tasks and gateways is supported. However, this comes with an additional requirement with respect to modelling the diagram. Please find below the modelling requirements.
```
Comment (TimKam) [inline README.md:11]: You can remove the "however"

File: src/models/models.controller.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export let bpmn2sol = xml =>
     let abi = "";
     const cont = parseModel(modelInfo);
     return cont.then(() => {
-      const input = {};
+      let input = {};
```
Comment (TimKam) [inline src/models/models.controller.ts:14]: Does this really need to be a `let`? We can change the object's properties in a `const`, we just can't change the pointer, right?

**Review by TimKam — APPROVED** [review]
Approved with nits. I merely _read_ the code. I don't know why the template files can be deleted, but I trust you.

File: README.md (hunk 1 above)
Comment (Matwog) [reply README.md:11]: Sure

File: src/models/models.controller.ts (hunk 2 above)
Comment (Matwog) [reply src/models/models.controller.ts:14]: Yes I was trying few other things with `input` so changed to `let` but forgot to revert back. Will change it 👍

**Pull request merged**